Gaijinizer patch/Map047.txt

```diff
@@ -569,100 +578,100 @@ struggle on the Minn front.#MTLed
 『そういう悪魔』が集まってる場所に遅れて目覚めて
  しかも、『トンカツ』なんだから！
 > CONTEXT: Map047/events/38/pages/0/365/Dialogue/hyakki
-I say, \\N[1] was unlucky yo! 
-I wake up late to the place where such demons
-gather. And because it's a "pork cutlet"!#MTLed
+I mean, \\N[1], you're just kinda unlucky! 
+Waking up in a place full of "those demons", and
+basically being like a "pork cutlet" to them!#Ron
 > END STRING
 
 > BEGIN STRING
 ただ……まだこの周辺にも、『血気盛ん』な奴は残ってる。
 
  上、見てみなヨ。
 > CONTEXT: Map047/events/38/pages/0/371/Dialogue/hyakki
-However...there are still "bloody" people
-around here. Above, look at everyone.#MTLed
+But... there's still some of those "hotblooded"
+types left around here. Take a gander above ya.#Ron
 > END STRING
 
 > BEGIN STRING
 上空には、ハゲワシほどの大きさの異形の鳥が
 
 電線やビルの屋上を群れるように飛び交っていた。
 > CONTEXT: Map047/events/38/pages/0/384/Dialogue
-In the upper sky, a deformed bird about the size of a
-vulture. It flew like a flock of wires and the roof of a
-building.#MTLed
+In the sky, deformed birds about the size of a
+vulture. They were flying around in a flock over
+the power lines and rooftops.#Ron
 > END STRING
 
 > BEGIN STRING
 ギャアギャアうるさいカラスの声がすると思っていたが
 
 どうやらこれも『悪魔』だったらしい……。
 > CONTEXT: Map047/events/38/pages/0/388/Dialogue
-* grumble * I thought that the voice of a loud crow
-Apparently this was also a "demo" ....#MTLed
+They sounded like noisy squawking crows to me,
+but they were apparently demons...#Ron
 > END STRING
 
 > BEGIN STRING
 あ、あれは…
 > CONTEXT: Map047/events/38/pages/0/392/Dialogue/kazuya
-Oh, that's ...#MTLed
+T- that's...#Ron
 > END STRING
 
 > BEGIN STRING
 『ハーピーモドキ』っていうヤツらなんだけどね。
 
 コイツらの主食は人間の精気じゃなくて血肉なんだヨ。
 > CONTEXT: Map047/events/38/pages/0/394/Dialogue/hyakki
-It's a guy called Beetle Harpy.
-The staple food of Koitsu et al.#MTLed
+Those guys are called "Harpy Things". They don't
```

Comment:
I had a fairly good idea about what to call them.... Would "Harpy Spawn" be a valid translation? I'll admit that I'm not the best translator, but it seems more thematic then "Thing".

Reply:
The nuance I'm picking up with that word and the way their appearance and behaviour is described in-game, is that they're an evil imitation of what a harpy's supposed to look like, and a twisted version of how harpies are supposed to behave. Acting more like a combination of hyena and vulture.

Based on that I'd say "Harpy Spawn" doesn't quite fit the translation, but thematically it feels appropriate as a kind of disdainful name for them.